runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchTransformTranslators.java

```diff
@@ -422,6 +423,38 @@ public void translateNode(
     }
   }
 
+  private static class ReshuffleKeysTranslatorBatch<K, InputT>
+      implements FlinkBatchPipelineTranslator.BatchTransformTranslator<Reshuffle.Keys<K, InputT>> {
+
+    @Override
+    public void translateNode(
```

Comment:
Could we avoid the copy&paste here and wrap the code into a reusable utility class instead?

Reply:
Updated and moved duplicate code in shared method/util.